File: C++ Code/Basics/Mensuration/2 D/Circle.cpp
// WAP to calculate area and perimeter of a circle,radius is given by user.
#include<iostream>
#define PI 3.14
using namespace std;
int main(){
    float r;
    cout<<"Using pie =3.14"<<endl<<"Enter the radius : ";
    cin>>r;
    cout<<endl<<"Area = "<<PI*r*r<<endl;
    cout<<"Perimeter = "<<2*PI*r;
    return 0;
}

File: Java Code/Basics/Mensuration/Circle.java
import java.util.*;

public class Circle{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the radius : ");
    float r=sc.nextFloat();
    System.out.println("For Circle (r = "+r+")");
    float P=44*r/7;
    float A=22*r*r/7;
    System.out.println("Area = "+A);
    System.out.println("Perimeter = "+P);
    System.out.println("\nFor Semi-Circle (r = "+r+")");
    A=A/2;
    P=P/2+r;
    System.out.println("Area = "+A);
    System.out.println("Perimeter = "+P);
  }
}